stdlib/REPL/src/docview.jl

```diff
@@ -402,8 +402,9 @@ function symbol_latex(s::String)
 
     return get(symbols_latex, s, "")
 end
-function repl_latex(io::IO, s::String)
-    latex = symbol_latex(s)
+function repl_latex(io::IO, s0::String)
+    s::String = s0
```

Comment:
This seems really mysterious to me, maybe add a comment? (sorry if this is obvious, I'm not up-to-date with this kind of improvements :) )